src/atc/functions.py

```diff
@@ -2,10 +2,47 @@
 A collection of useful pyspark snippets.
 """
 
+from atc.spark import Spark
 from pyspark.sql.functions import udf
 from pyspark.sql.types import StringType
 import uuid as _uuid
+import pyspark.sql.functions as F
+from pathlib import Path
 
 # Pyspark uuid function implemented as recommended here
 # https://stackoverflow.com/questions/49785108/spark-streaming-with-python-how-to-add-a-uuid-column/50095755
 uuid = udf(lambda: str(_uuid.uuid4()), StringType()).asNondeterministic()  # noqa
+
+def init_dbutils(spark):
+    try:
+        from pyspark.dbutils import DBUtils
+        dbutils = DBUtils(spark)
+    except ImportError:
+        import IPython
+        dbutils = IPython.get_ipython().user_ns["dbutils"]
+    return dbutils
+
+def drop_table_cascade(DataBaseName,TableName):
```

Comment:
You only use DataBaseName and TableName in the combination `f"{DataBaseName}.{TableName}"`. Is it not ok to just give this as a single argument?